src/cashier/withdraw.es6

```diff
@@ -7,12 +7,11 @@ import liveapi from 'websockets/binary_websockets';
 import windows from 'windows/windows';
 import rv from 'common/rivetsExtra';
 import currencyDialog from 'cashier/currency';
-import _ from 'lodash';
+import {debounce} from 'lodash'
```

Comment:
@arnabk It turns out that the syntax for loading a specific function from lodash is as described above. `import debounce from 'lodash/debounce'` doesn't work.